python/hsfs/engine/python.py

```diff
@@ -670,6 +670,24 @@ def write_training_dataset(
                 "Currently only query based training datasets are supported by the Python engine"
             )
 
+        if (
+            arrow_flight_client.get_instance().is_query_supported(
+                dataset, user_write_options
+            )
+            and len(training_dataset.splits) == 0
+            and len(training_dataset.transformation_functions) == 0
```

Comment:
does flying duck support output format other than parquet?

Reply:
Not yet, I should filter that as well